tools/python/maps_generator/generator/diffs.py

```diff
@@ -32,8 +32,12 @@ def calculate_diff(params):
     if out.exists():
         status = Status.NOTHING_TO_DO
     else:
-        res = subprocess.run("mwm_diff_tool " + old.as_posix() + " " + new.as_posix() + " " + out.as_posix())
+        # FIXME: running as a single string without shell=True doesn't work (did it ever?) and mwm_diff_tool is actually pymwm_diff CMAKE target
+        res = subprocess.run(["mwm_diff_tool", "make", old.as_posix(), new.as_posix(), out.as_posix()],
+            stderr=subprocess.STDOUT, stdout=subprocess.PIPE)
```

Comment:
I'm not sure what this used to do, but I think it should be using [this tool](https://github.com/organicmaps/organicmaps/blob/b600aafa24db9baf629125ee8d9cb06609c5847a/generator/mwm_diff/mwm_diff_tool/CMakeLists.txt#L1) which gets built as `pymwm_diff`. Would it make sense to update the binary name to `mwm_diff_tool`?